src/test/resources/emails/instructorNewAccountRequestAcknowledgementEmailWithComments.html

```diff
@@ -0,0 +1,65 @@
+<p>Hello, Darth Vader</p>
+
+<p>
+  Thank you for submitting an account request. This is what you have submitted:
+</p>
+
+<div>
+  <table style="max-width:600px;border:1px solid black;">
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Full Name
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        Darth Vader
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Country & Institute
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        Sith Order
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Email Address
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        darth-vader@sith.org
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Home Page URL<br>& Comments
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        I Am Your Father
+      </td>
+    </tr>
+  </table>
+</div>
+
+<p>
+  Your request will be reviewed within 24 hours. We will send another email once your request has been accepted.
+</p>
+<p>
+  If you have any additional queries, please feel free to contact us at app_admin@gmail.com.
```

Comment:
```suggestion
  If you have any additional queries, please feel free to contact us at ${support.email}.
```

The ones in the test cases should also use `${support.email}` instead. In the test cases, `EmailChecker` should replace it accordingly. You can double check this.

Just in case, don't worry over the fact that the placeholder you already have in your template `${supportEmail}` is slightly different from the placeholder for the test cases `${support.email}`; `${supportEmail}` seems to be the convention in the email generator so that's fine. However, the one for the test cases should be `${support.email}` because it should be the one used by `EmailChecker`.